### DISCUSSION TITLE
Data Sets

### DISCUSSION THREAD

**Discussion by efolsom17** [efolsom17/Spring2023-MATH447-Project#1] [Ideas]
Any Data sets here can be linked

**Comment by greend20** [comment]
[lovoo_v3_users_instances.csv](https://github.com/efolsom17/Spring2023-MATH447-Project/files/11226033/lovoo_v3_users_instances.csv)
[lovoo_v3_users_instances.csv](https://github.com/efolsom17/Spring2023-MATH447-Project/files/11226035/lovoo_v3_users_instances.csv)

**Reply by greend20** [reply]
Content : The dataset contains user profile infos of users of the website [Lovoo](https://lovoo.com/).

The dataset was gathered during spring 2015 (april, may). At that time, Lovoo was expanding in european countries (among others), while Tinder was trending both in America and in Europe.
At that time the iOS version of the Lovoo app was in version 3.

**Comment by greend20** [comment]
[golf_df.csv](https://github.com/efolsom17/Spring2023-MATH447-Project/files/11226064/golf_df.csv)

**Reply by greend20** [reply]
Content: Smaller data set on golf plays and various weather factors

**Comment by greend20** [comment]
https://www.kaggle.com/code/zikazika/analysis-of-world-crime
Content: example of how someone scraped crime data off the internet to compare crimes across the world as well as by state for America. They also included code of how they analyzed their gathered data. If we wanted to scrape any data I thought this was a good example.

**Comment by greend20** [comment]
[Datasets - Book Exercises (1).xlsx](https://github.com/efolsom17/Spring2023-MATH447-Project/files/11226135/Datasets.-.Book.Exercises.1.xlsx)
[Datasets - Book Exercises.xlsx](https://github.com/efolsom17/Spring2023-MATH447-Project/files/11226136/Datasets.-.Book.Exercises.xlsx)

Content: This is an attempt to figure out if books can be judged by the blurb or cover. I wanted to find out if the description of a book is indicative of the genre or the rating of the book. Or if the genre of a book can be figured out by looking at the cover.

Note- I tried to paste the Csv file on here but it's too large. If we decide on this dataset I do have the csv file we could upload to R.

**Comment by efolsom17** [comment]
https://www.kaggle.com/datasets/rdoume/beerreviews
https://www.kaggle.com/datasets/schmoyote/coffee-reviews-dataset
https://www.kaggle.com/datasets/nvisagan/cannabis-strains-features
https://www.kaggle.com/datasets/whenamancodes/speed-dating
https://www.kaggle.com/datasets/surajjha101/myntra-reviews-on-women-dresses-comprehensive

Sent by Eve
